app/controllers/measurements_controller.rb

```diff
@@ -17,7 +17,7 @@ def aggregated
     @measurements = @measurements.group('DATE(created_at)').reorder('DATE(created_at)')
     @minimum_temperature = @measurements.minimum(:temperature)
     @maximum_temperature = @measurements.maximum(:temperature)
-    @average_temperature = @measurements.average(:temperature)
+    @average_temperature = @measurements.average(:temperature).transform_values(&:to_f)
```

Comment:
Well… this workaround results out of a 12 years old specialty of `AVG`: https://github.com/rails/rails/pull/40351

Reply:
Haha, nice work digging into that old code :slightly_smiling_face: